docs/essentials/datetime/ConvertingBetweenTimeZones.md

```diff
@@ -1 +1,99 @@
+# Converting Times Between Time Zones
+
+It is becoming increasingly important for any application that works with dates and times to handle differences between time zones. An application can no longer assume that all times can be expressed in the local time, which is the time available from the [System.DateTime](http://dotnet.github.io/api/System.DateTime.html) structure. For example, a Web page that displays the current time in the eastern part of the United States will lack credibility to a customer in eastern Asia. This topic explains how to convert times from one time zone to another, as well as how to convert `DateTimeOffset` values that have limited time zone awareness.
```

Comment:
> It is becoming increasingly important for any application that works with dates and times to handle differences between time zones

We should assume that the point of importance has passed. It's no longer "increasingly important", just "important".